console/src/main/scripts/plugins/accessPortal/vendor/js/redhat_access_angular_ui.js

```diff
@@ -3522,7 +3522,7 @@ angular.module('RedhatAccess.cases').controller('EditGroup', [
         $scope.accountNumber = null;
         $scope.isUsersPrestine = true;
```

Comment:
Ultimately, lets get rid of this $scope syntax and use 'ControllerAs' syntax instead.